### PR TITLE
Update DG & UG

### PR DESCRIPTION
**Pull request by bharathcs**
- Remove trailing whitespaces that I did just now :(
- ~~Add python script for picking up headings~~ (reverted)
- Update DG and UG.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -248,12 +247,29 @@ remove an existing asset/liability as shown [here](#4331-add-an-asset-to-a-clien
...
+type into it, and all the edits you make will be saved. Unlike the command box, this supports multi-line input and
+allows you to indent lines with spaces.
 
 ![Notes Tab of DonnaFin](images/Notes.png)
 
 ### 3.4 Command Format
 
+In DonnaFin, we provide the user with a Command-Line-Input (or CLI). The command message is generally in this format:
+
+<span style="color: red">COMMAND_WORD</span><span style="color: blue"> ARG1 ARG2 ARG3</span>
```
Comment (bharathcs) [inline docs/UserGuide.md:260]:
```suggestion
In DonnaFin, we provide the user with a Command-Line-Input (or CLI). The command message is generally in the format of a `COMMAND_WORD` followed optionally by a number of arguments (`COMMAND_WORD ARG1 ARG2 ARG3`).
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -435,7 +457,8 @@ Examples:
 
 #### 4.2.6 Clearing all client data : `clear`
 
-Clears all data from the DonnaFin.
+Clears all data from the DonnaFin. This is a destructive command that cannot be recovered from, please take care when
```
Comment (sheehui) [inline docs/UserGuide.md:460]: can use the caution block for "This is a destructive command that cannot be recovered from, please take care when..."

**Review by sheehui — COMMENTED** [review]
love ur improvements

File: docs/UserGuide.md [hunk 3]
```diff
@@ -177,7 +177,7 @@ your client:
 3. Email
 4. Address
 
-This information is represented as shown in the picture below.
+The picture below shows how these attributes are displayed.
```
[the remark this answers is not in the document]
Comment (bharathcs) [reply docs/UserGuide.md:180]: good call, changed for all UG refs to attributes.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -203,13 +202,14 @@ Here are the following fields that each policy records:
 4. Yearly Premium
 5. Commission
 
-Each policy has an index which is sorted by Ascending order, and if you would like to remove an existing policy, you can
-keep track of the index by looking at the number to the left of the policy name you would like to delete.
+Each policy has an index which is sorted by ascending order, and if you would like to remove an existing policy, you can
+use its index as an argument for delete. To save you the effort of counting rows, we have indexed the rows with a counter
```
[the remark this answers is not in the document]
Comment (bharathcs) [reply docs/UserGuide.md]: Yep, I get that. Fixed to parameter.

File: docs/UserGuide.md (hunk 2 above)
Comment (bharathcs) [reply docs/UserGuide.md]: Done!

**Review by sheehui — APPROVED** [review]
lgtm! :))))

**Pull request merged**